src/main/java/moli/Moli.java

```diff
@@ -0,0 +1,196 @@
+package moli;
+
+import moli.task.Task;
+import moli.task.Todo;
+import moli.task.Deadline;
+import moli.task.Event;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Moli {
+    private static final int MAX_TASKS = 100;
+    private static final ArrayList<Task> tasks = Storage.loadTasks(); // Load tasks from file
+    private static final String DIVIDER = "____________________________________________________________";
+
+    public static void main(String[] args) {
```

Comment:
This function is >30 LOC, which is a little long. Consider refactoring the inner sections of this method into other methods.